scripts/validate-helm.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+export KUBERAY_HOME=$(git rev-parse --show-toplevel)
+SCRIPT_PATH="${KUBERAY_HOME}/scripts/openapi2jsonschema.py"
+RAYCLUSTER_CRD_PATH="$KUBERAY_HOME/ray-operator/config/crd/bases/ray.io_rayclusters.yaml"
+tmp=$(mktemp -d)
+trap 'rm -rf "$tmp"' EXIT
+
+# Convert CRD YAML to JSON Schema
+pushd "${tmp}" > /dev/null
+chmod +x "$SCRIPT_PATH"
```

Comment:
This line can be removed because now this file is stored in our repo and it is already an executable.